packages/@sanity/cli/src/actions/init-project/templates/appQuickstartSanityUi.ts

```diff
@@ -0,0 +1,32 @@
+import {type ProjectTemplate} from '../initProject'
+
+const appSanityUiTemplate: ProjectTemplate = {
+  dependencies: {
+    '@sanity/sdk': '^0.0.0-rc',
+    '@sanity/sdk-react': '^0.0.0-rc',
+    '@sanity/ui': '^2.15.12',
```

Comment:
Should this be similar to sanity? 

```suggestion
    '@sanity/ui': '^2',
```